client/components/ZoopSentDialog.tsx

```diff
@@ -0,0 +1,44 @@
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import Link from "@mui/material/Link";
+import React, {useState} from "react";
+import { useNavigate } from "react-router-dom";
+import { Zoop } from "../../src/types/custom";
+
+
+export interface ZoopSentDialogProps {
+    sentZoop: Zoop
+    onClose: () => void
+}
+
+const ZoopSentDialog = ({sentZoop, onClose}: ZoopSentDialogProps) => {
+    
+    const navigate = useNavigate()
+
+    return (
+        <Dialog
+            open={true}
+            onClose={onClose}
+        >
+            <DialogTitle>Zoop Sent!</DialogTitle>
+            <DialogContent>
+                <DialogContentText> 
+                    <Link
```

Comment:
Don't recommend using a Link with navigate.

Prefer using React Router link (using component prop). Check out the example in Robot Pet City to see how to make TS happy

Reply:
Out of curiosity, is there a reason to use React Router link over MUI's Link with useNavigate?

Reply:
In my experience when a link has an onClick most libraries no longer treat it as a link but as a button. So HTML would not be semantic.

Also HTML link has a URL location, which is what React Router link will create for you. So it is more easy for users to tell where the link will go. It will behave like a link (eg. Showing the link in bottom of browser window while hovering, or maybe reading where the URL will be for some screen readers depending on settings).

So those are my instincts, since MUI has no idea what React Router is doing (eg. What navigate function is) I believe this leads to less semantic and less accessible markup.